src/frontend_workspaces/agentic_chat/src/PoliciesConfig.css

```
@@ -3,3 +3,24 @@
+.policy-code-block pre {
+  margin: 0;
+  padding: 1rem;
+  overflow-x: auto;
+  white-space: pre;
+  line-height: 1.6;
+  background: #161616;
+  border-radius: 4px;
+}
+
+.policy-code-block pre code {
+  font-family: "IBM Plex Mono", "Consolas", monospace;
```

Stylelint (17.14.0)

[error] Expected no quotes around "Consolas" (font-family-name-quotes)

(font-family-name-quotes)